src/client.js

```diff
@@ -476,6 +476,113 @@ class ConnectionParameters {
 
 }
 
+// ========================================================================================
+// File Uploader
+// ========================================================================================
+
+const fileUploader = (client) => {
+  /**
+   * Will increament the counter
+   * @returns {Promise<number|*>}
+   */
+  const increaseValue = async () => {
```

Comment:
Private functions should have “_” prefix (increaseValue, etc.)
If not, they are considered public, and you'll have to maintain their signature and compatibility forever